oidc_cli/oidc_impl/client/client.go

```diff
@@ -36,7 +36,7 @@ type Config struct {
 }
 
 // NewClient returns a new client
-func NewClient(ctx context.Context, config *Config, clientOptions ...Option) (*Client, error) {
+func NewClient(ctx context.Context, config *Config, scopes []string, clientOptions ...Option) (*Client, error) {
```

Comment:
remove this scopes argument and use the clientOptions instead

Reply:
I'm trying to understand how this option works. Could you explain how we'd use it?

Reply:
kind of like this comment: https://github.com/chanzuckerberg/go-misc/pull/1082#discussion_r1887672123.

You'd have an Option function that updates the scopes, rather than passing them in.